GameServerLib/GameObjects/Other/Stackable.cs

```diff
@@ -19,14 +19,27 @@ public virtual bool IncrementStackCount()
             return true;
         }
 
-        public virtual bool DecrementStackCount()
+        public virtual bool DecrementStackCount(int stacksToRemove = 0)
```

Comment:
Revert and use SetStacks instead.

Reply:
Using SetStacks conflicts with the `if (newStacks < 1 || newStacks > MaxStacks)` when setting the stacks to 0 in order to remove the item, i'd have to delete that, if that's alright

Reply:
Simply check in RemoveItem before calling SetStacks.

Reply:
Resolved